python/setup.py

```diff
@@ -146,6 +146,11 @@ def find_version(*filepath):
     "six >= 1.0.0",
     "flatbuffers",
     "faulthandler;python_version<'3.3'",
+    # simplejson and psutil are required by the Reporter.
+    "simplejson",
```

Comment:
What are the advantages of this over `json`? We're already using `json` in a bunch of places, and it'd be great to keep the number of `json` libraries to a minimum.

Reply:
simplejson has `namedtuple_as_object` which serialises psutil's namedtuples as a dict with keys->values, I'm not aware of a simple method that exists to do this in the standard module, though a custom encoder can be written, I can go ahead and do that.

Reply:
Created a recursive method that works for this simple case.